### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
# Description
Updated EN -> JA documentation translations

English texts exist in this PR. Please leave them as they are. (They will be translated next time.)

### CODE REVIEW

**Review by rosieyohannan — APPROVED** [review]
🎉

File: jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc [hunk 1]
```diff
@@ -1,136 +1,155 @@
...
+. 最後に、TCP ポート 80 (つまり HTTP) が開いていて アクセス可能であることを確認します。 セキュリティーを強化するために、Nomad クライアントと `machine` とリモート Docker Engine の VM にのみポートを開くことをお勧めします。
 +
-たとえば AWS では、セキュリティ グループと OS レベルのファイアウォールの両方が HTTP 経由で Nomad クライアントからの接続を受け付けるように正しく設定されていることを確認する必要があります。
+たとえば AWS では、セキュリティグループと OS レベルのファイアウォールの両方が Nomad クライアントからの接続を HTTP 経由で受け入れるように正しく設定されていることを確認する必要があります。
 
-=== Set Up a Secure Pull Through Cache Registry (HTTPS Proxy for Docker Hub)
+=== セキュアなプルスルーキャッシュレジストリ (Docker Hub の HTTPS プロキシ) の設定
 
-Under certain circumstances (especially when you are enabling https://docs.docker.com/develop/develop-images/build_enhancements/[BuildKit] on `machine` executors and remote Docker engines), your pull through cache registry needs to accept connections over HTTPS. These instructions cover setting up a pull through cache that listens for HTTPS connections.
+特に `machine` Executor とリモート Docker Engine で https://docs.docker.com/develop/develop-images/build_enhancements/[BuildKit] を有効にしている場合などの特定の状況において、プルスルーキャッシュレジストリが HTTPS での接続を受け入れる必要がある場合があります。  ここでは、HTTPS 接続をリッスンするプルスルーキャッシュの設定方法を紹介します。
```
Comment (jasurbek-khanjarov) [inline jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc:74]: https://docs.docker.com/develop/develop-images/build_enhancements/[BuildKit] =>
[BuildKit](https://docs.docker.com/develop/develop-images/build_enhancements/)

File: jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc [hunk 2]
```diff
@@ -1,136 +1,155 @@
...
+  -e REGISTRY_PROXY_REMOTEURL="https://registry-1.docker.io" \
+  -e REGISTRY_PROXY_USERNAME=DOCKER_HUB_USERNAME \
+  -e REGISTRY_PROXY_PASSWORD=DOCKER_HUB_ACCESS_TOKEN \
+  -e REGISTRY_HTTP_TLS_CERTIFICATE=/data/tls/fullchain.pem \
+  -e REGISTRY_HTTP_TLS_KEY=/data/tls/privkey.pem \
+  registry
 ----
 
-. Finally, *make sure that the TCP port 443 (i.e. HTTPS) is open and accessible*. For better security, we recommend you to open the port only to Nomad clients and VMs for `machine` executors and remote docker engines.
+. Finally, *make sure that the TCP port 443 (i.e. HTTPS) is open and accessible*. セキュリティーを強化するために、Nomad クライアントと `machine` とリモート Docker Engine の VM にのみポートを開くことをお勧めします。
```
Comment (jasurbek-khanjarov) [inline jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc:111]: There is no translated part.

File: jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc [hunk 3]
```diff
@@ -184,44 +203,41 @@ jobs:
...
-. Run `sudo docker restart vm-service` to apply the change.
+. `sudo docker restart vm-service` を実行して変更を適用します。
 
-== Resources
+== 関連資料
 
-* https://docs.docker.com/registry/recipes/mirror/[How to configure a pull through cache mirror]
-* https://hub.docker.com/_/registry[Official Docker Registry Docker image]
-* https://docs.docker.com/registry/configuration/[How to configure official Docker Registry]
+* https://docs.docker.com/registry/recipes/mirror/[プルスルーキャッシュミラーの設定方法]
```
Comment (jasurbek-khanjarov) [inline jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc:241]: https://docs.docker.com/registry/recipes/mirror/[プルスルーキャッシュミラーの設定方法] =>
[プルスルーキャッシュミラーの設定方法](https://docs.docker.com/registry/recipes/mirror/)

File: jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc [hunk 4]
```diff
@@ -184,44 +203,41 @@ jobs:
...
+* https://hub.docker.com/_/registry[公式 Docker レジストリの Docker イメージ]
```
Comment (jasurbek-khanjarov) [inline jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc:242]: https://hub.docker.com/_/registry[公式 Docker レジストリの Docker イメージ] =>
[公式 Docker レジストリの Docker イメージ](https://hub.docker.com/_/registry)

File: jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc [hunk 5]
```diff
@@ -184,44 +203,41 @@ jobs:
...
+* https://docs.docker.com/registry/configuration/[公式 Docker レジストリの設定方法]
```
Comment (jasurbek-khanjarov) [inline jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc:243]: https://docs.docker.com/registry/configuration/[公式 Docker レジストリの設定方法] =>
[公式 Docker レジストリの設定方法](https://docs.docker.com/registry/configuration/)

File: jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc [hunk 6]
```diff
@@ -1,136 +1,155 @@
...
 
-ここでは、Docker Hub のレジストリのミラーを設定する方法について説明します。
+ここでは、Docker Hub プルスルーレジストリミラーを設定する方法について説明します。
 
 toc::[]
 
 == 概要
 
-https://www.docker.com/blog/scaling-docker-to-serve-millions-more-developers-network-egress/[2020年11月1日]から、Docker Hub で匿名ユーザーに対する発信元 IP に基づいたプル回数制限が導入されます。 運用中のサービスでの混乱を避けるために、Docker Hub から認証付きでプル操作を行うようにすることをお勧めします。 認証は設定ファイルで行えます(https://circleci.com/ja/docs/2.0/private-images/[Docker の認証付きプルの使用]を参照)。
+https://www.docker.com/blog/scaling-docker-to-serve-millions-more-developers-network-egress/[2020 年 11 月 1 日]より、Docker Hub で匿名ユーザーに対する発信元 IP に基づいたプル回数制限が導入されています。 運用中のサービスでの混乱を避けるために、Docker Hub から認証付きでプル操作を行うようにすることをお勧めします。 認証は設定ファイルで行えます (https://circleci.com/ja/docs/2.0/private-images/[Docker の認証付きプルの使用]を参照)。
```
Comment (jasurbek-khanjarov) [inline jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc:22]: (https://circleci.com/ja/docs/2.0/private-images/[Docker の認証付きプルの使用]を参照) =>
[Docker の認証付きプルの使用](https://circleci.com/ja/docs/2.0/private-images/)を参照

**Review by jasurbek-khanjarov — COMMENTED** [review]
There were some broken links with the wrong markdown. I suggested the fixes but let me know if I am missing something.

File: jekyll/_cci2_ja/bitbucket-integration.adoc [hunk 7]
```diff
@@ -81,9 +81,9 @@ When you add a new project, CircleCI creates a deployment key on Bitbucket for y
 
 **What is a user key?**
 
-A user key is user-specific an SSH key-pair. Bitbucket stores the public key, and CircleCI stores the private key. 秘密鍵を持っていると、プロジェクトへの「Git」アクセスの目的で、そのユーザーとして行動することができます。
+A user key is user-specific an SSH key-pair. Bitbucket stores the public key, and CircleCI stores the private key. プライベートキーを持っていると、プロジェクトへの「Git」アクセスの目的で、そのユーザーとして行動することができます。
```
[the remark this answers is not in the document]
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/bitbucket-integration.adoc:84]: Yes! A translation vendor is now working on the English part, will be applied next week.

File: jekyll/_cci2_ja/webhooks.md [hunk 8]
```diff
@@ -45,12 +45,12 @@ CircleCI は、Webhook に応答したサーバーが 2xx のレスポンス コ
...
-| ------------------- | --------------------------------------------------------------------------------------- |
-| 型                   | `application/json`                                                                      |
-| User-Agent          | 送信者が CircleCI であることを示す文字列（`CircleCI-Webhook/1.0`）。 この値はプレビュー期間中に変更される場合があります。 |
-| Circleci-Event-Type | イベントのタイプ （`workflow-completed`、`job-completed`など）                                       |
-| Circleci-Signature  | この署名により Webhook の送信者にシークレット トークンへのアクセス権が付与されているかどうかを検証することができます。                        |
+| ヘッダー名               | 値                                                                             |
+| ------------------- | ----------------------------------------------------------------------------- |
+| 型                   | `application/json`                                                            |
+| User-Agent          | 送信者が CircleCI であることを示す文字列（`CircleCI-Webhook/1.0`)。 この値はプレビュー期間中に変更される場合があります。 |
+| Circleci-Event-Type | イベントのタイプ （`workflow-completed`、`job-completed`など）                             |
```
[the remark this answers is not in the document]
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/webhooks.md:52]: I think unnecessary spaces were just removed on the English file, so we don't have to worry about it.

File: jekyll/_cci2_ja/add-ssh-key.md [hunk 9]
```diff
@@ -72,7 +72,10 @@ version:
 
 すべての CircleCI ジョブは、`ssh-agent` を使用して登録済みのすべての SSH キーに自動的に署名します。 ただし、コンテナに実際にキーを登録するには、`add_ssh_keys` キーを**必ず使用してください**。
 
-SSH キーをコンテナに追加するには、 [特別なステップ]({{ site.baseurl }}/ja/2.0/configuration-reference/#add_ssh_keys) である`add_ssh_keys` を設定ファイルの適切な [ジョブ]({{ site.baseurl }}/ja/2.0/jobs-steps/) の中で使用します。
+SSH キーをコンテナに追加するには、 [特別なステップ]({{site.baseurl}}/ja/2.0/configuration-reference/#add_ssh_keys) である`add_ssh_keys` を設定ファイルの適切な [ジョブ]({{ site.baseurl }}/ja/2.0/jobs-steps/) の中で使用します。
+
+セルフホストランナーの場合、システムに `ssh-agent`  があり`add_ssh_keys` ステップが正常に使用できることを確認して下さい。 SSH キーは、`$HOME/.ssh/id_rsa_<fingerprint>`に記述されます。`$HOME`は、ジョブを実行するように設定されたユーザーのホームディレクトリで、`<fingering>` はこのキーのフィンガープリントです。 ホストエントリーは、キーを使用するための関連する `IdentityFile` オプションと一緒に `$HOME/.ssh/config` にも追加されます。
```
[the remark this answers is not in the document]
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/add-ssh-key.md:77]: Thank you! Applied.

File: jekyll/_cci2_ja/faq.md [hunk 10]
```diff
@@ -110,144 +192,102 @@ Docker Hub のパブリックイメージについては、下記のようにア
 my-user/couchdb:1.6.1
 ```
 
-### Docker イメージのバージョンを指定するときのベストプラクティスは？
+### Docker イメージのバージョンを指定するときのベストな方法は？
 {: #what-is-the-best-practice-for-specifying-image-versions }
 {:.no_toc}
-`latest` タグを **付けず** に Docker イメージを指定することをお勧めします。 もしくは、特定のバージョンやタグを付けるのも良い方法です。ベースとなるイメージのディストリビューションに変更があった際に、アップストリームの変更によるコンテナへの影響を防ぐには、例えば `circleci/ruby:2.4-jessie-node` のように指定します。 `circleci/ruby:2.4` とだけ指定した場合は、たとえば、`jessie` から `stretch` への予期しない変更による影響を受ける可能性があります。 他の応用例を知りたいときは、「Executor タイプの選び方」の[Docker イメージ活用のヒント]({{ site.baseurl }}/ja/2.0/using-docker/#docker-image-best-practices)や、「CircleCI のビルド済み Docker イメージ」の[ビルド済みイメージの活用方法]({{ site.baseurl }}/ja/2.0/circleci-images/#best-practices)を参照してください。
+`latest` タグを**付けず**に Docker イメージを指定することをお勧めします。 もしくは、特定のバージョンやタグを付けるのも良い方法です。ベースとなるイメージのディストリビューションに変更があったとき、イメージを固定し、コンテナへのアップストリームの変更を防ぐには、例えば `cimg/ruby:3.0.4-browsers` のように指定します。 例えば、`cimg/ruby:3.0.4` のみを指定した場合、`browsers` から `node` に予期せぬ変更が加えられる場合があります。 その他の応用例は、[Docker イメージのベストプラクティス]({{site.baseurl}}/2.0/using-docker/#docker-image-best-practices)や[CircleCI イメージのベストプラクティス]({{site.baseurl}}/ja/2.0/circleci-images/#best-practices)を参照してください。
```
[the remark this answers is not in the document]
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/faq.md:198]: Thank you! Applied.

File: jekyll/_cci2_ja/faq.md [hunk 11]
```diff
@@ -110,144 +192,102 @@ Docker Hub のパブリックイメージについては、下記のようにア
...
+- [iOS]({{site.baseurl}}/ja/2.0/ios-tutorial/)
+- [Windows]({{site.baseurl}}/ja/2.0/using-windows/)
 
-### スケジュールされたワークフローは、指定された時間通り正確に実行されますか？
-{: #are-scheduled-workflows-guaranteed-to-run-at-precisely-the-time-scheduled }
+### CircleCI がサポートしている CPU アーキテクチャは？
+{: #which-cpu-architectures-does-circleci-support }
 {:.no_toc}
-スケジュールの正確性については保証できません。 設定した時間にコミットがプッシュされたとしてワークフローをスケジュール実行します。
+Docker ジョブでは `amd64`を、マシンジョブでは `amd64` と [ARM リソース]({{site.baseurl}}/ja/2.0/arm-resources/) をサポートしています。
```
[the remark this answers is not in the document]
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/faq.md:247]: Thank you! Applied.

File: jekyll/_cci2_ja/faq.md [hunk 12]
```diff
@@ -256,26 +296,26 @@ CircleCI Server v３.x および v2.x で Windows Exexutor をご利用いただ
 
 必要に応じて、並列実行を使用してビルド時間をさらに短縮できます。並列実行を使用すると、ジョブを複数のテストに分割して同時に実行できます。 2倍の並列実行により、通常 2,500 分で実行されるビルドが 1,250 分で実行できるため、開発者の生産性がさらに向上します。 2つの Executor がそれぞれ 1,250 分間並行して実行している場合、合計ビルド時間は 2,500 分になります。
 
-#### 異なる組織間で契約プランを共有できますか？ その場合、請求を 1 箇所にまとめることは？
+### 異なる組織間で契約プランを共有できますか？ その場合、請求を 1 箇所にまとめることは？
 {: #is-there-a-way-to-share-plans-across-organizations-and-have-them-billed-centrally }
 {:.no_toc}
-可能です。CircleCI アプリのサイドバーから `Plan` を選択し、`Share & Transfer` をクリックします。
+選択したクレジット パッケージの料金が、毎月初めに請求されます。
```
[the remark this answers is not in the document]
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/faq.md:302]: Thank you! This came from wrong translation memory...I've fixed it.

File: jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc (hunk 2 above)
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/docker-hub-pull-through-mirror.adoc:111]: Thank you! This part will be updated next week.

**Review by jasurbek-khanjarov — APPROVED** [review]

**Pull request merged**